cookbook/configuration/pdo_session_storage.rst

```diff
@@ -172,10 +229,10 @@ following (MySQL):
 .. code-block:: sql
 
     CREATE TABLE `session` (
-        `session_id` VARBINARY(128) NOT NULL PRIMARY KEY,
-        `session_data` BLOB NOT NULL,
-        `session_time` INTEGER UNSIGNED NOT NULL,
-        `session_lifetime` MEDIUMINT NOT NULL
+        `sess_id` VARBINARY(128) NOT NULL PRIMARY KEY,
+        `sess_data` BLOB NOT NULL,
```

Comment:
I just had an issue where `BLOB` wasn't large enough for my needs. When the size is exceeded, the user is logged out without warning...

I had to change to `MEDIUMBLOB`. Should we make a note about this?